auth/user_mgt.go

```diff
@@ -416,6 +424,13 @@ func validatePhone(val interface{}) error {
 	return nil
 }
 
+func validateValidSince(val interface{}) error {
+	if _, ok := val.(int64); !ok {
+		return fmt.Errorf("tokens valid after time must be an integer")
```

Comment:
I would either write this as:
"TokensValidAfterTime must be an integer"
or
"valid-after time for tokens must be an integer"

As it is, it's hard to read, because it's not clear that "tokens valid after time" is a compound, especially since the field is called "validSince" in other parts of the API.

Actually, why is it "validSince" in some places and "TokensValidAfterTime" in another?

Reply:
This is part of what was decided in the node api.

Reply:
It is valid since internally. it is exposed as tokensValidAfterMillis